src/fake_salts.c

```diff
@@ -164,7 +164,7 @@ static char *LoadUserClass(char which, int i) {
 		loaded = 1;
 	}
 
-	if ((user_class = cfg_get_param("Regen_Salts_UserClasses", NULL, user_class_num))) {
+	if ((user_class = (char*)cfg_get_param("Regen_Salts_UserClasses", NULL, user_class_num))) {
 		extern char *userclass_expand(const char *src); /* found in rules.c */
 		return userclass_expand(user_class);
 	}
```

Comment:
Do we need the cast here? The context suggests that we don't.

Reply:
I misread the code. I'm reverting it, changing to `const char *user_class`.

BTW that external function declaration is local - is that OK? I recall someone mentioned it's "bad".

Reply:
I'm OK with us importing scope-local `extern` symbols for now. This can be even cleaner than having them at the top of the file, and thus unnecessary at global scope.

If we want to make things cleaner, we'd need to have all external function prototypes in header files, with no exceptions.